composer.json

```diff
@@ -23,7 +23,7 @@
     "doctrine/inflector": "^1.3",
     "mockery/mockery": "^1.2",
     "phpstan/phpstan": "^0.12",
-    "phpunit/phpunit": "^7.5",
+    "phpunit/phpunit": "^7.5|^9.3",
```

Comment:
Should be `||` as it's new standard ;)
See https://github.com/composer/composer/issues/6755#issuecomment-337588505

Reply:
corrected :)